boards/arduino-mega2560/include/periph_conf.h

```diff
@@ -264,23 +265,67 @@ extern "C" {
 /**
  * @brief GPIO configuration
  */
-#define GPIO_NUMOF          (0U)
-#define GPIO_0_EN           0
-#define GPIO_1_EN           0
-#define GPIO_2_EN           0
-#define GPIO_3_EN           0
-#define GPIO_4_EN           0
-#define GPIO_5_EN           0
-#define GPIO_6_EN           0
-#define GPIO_7_EN           0
-#define GPIO_8_EN           0
-#define GPIO_9_EN           0
-#define GPIO_10_EN          0
-#define GPIO_11_EN          0
-#define GPIO_12_EN          0
-#define GPIO_13_EN          0
-#define GPIO_14_EN          0
-#define GPIO_15_EN          0
+#define GPIO_NUMOF          12
+#define GPIO_2_EN           1
```

Comment:
It's because there are pin 2-13 at PWM- and 0-1 in communication-section on the pinout.
for example: if you want to set pin 2, you have to call the method with 2 (parameter: dev).
But when you change the pin_map, that GPIO_0 is first, you will have the wrong pin all time, so i decided to implement it this way.